ftdetect/markdown.vim

```diff
@@ -1,3 +1,3 @@
 " markdown filetype file
-au BufRead,BufNewFile *.{md,mdown,mkd,mkdn,markdown,mdwn}   set filetype=mkd.markdown
```

Comment:
@cirosantilli yeah, `preserve` or `keep`. `set ft=mkd.markdown` tells vim to look for both `syntax/mkd.vim` and `syntax/markdown.vim` in its `runtimepath`. But if we change the name of syntax file from `mkd.vim` to `markdown.vim`, there's no way for vim to find the `syntax/mkd.vim`. Then it would be meaningless to `set ft=mkd.markdown` because only the markdown part of it works.